src/olympia/addons/serializers.py

```diff
@@ -1562,20 +1569,25 @@ def fake_object(self, data):
             # set .approved_for_groups cached_property because it's used in
             # .approved_applications.
             approved_for_apps = promoted.get('approved_for_apps')
-            obj.promoted = PromotedAddon(
-                addon=obj,
-                approved_application_ids=approved_for_apps,
-                created=None,
-                group_id=promoted['group_id'],
-            )
+            obj.promoted = [
```

Comment:
This requires an update to the docs: https://mozilla.github.io/addons-server/topics/api/overview.html#v5-api-changelog